lib/client/signalflow/request_manager.js

```diff
@@ -390,11 +390,34 @@ function RequestManager(options) {
     //destroy all open SSE connections
   }
 
+  function registerLiveTail(callbacks, overrideRequestId ){
+    var requestId = overrideRequestId || getRequestIdAndIncrement();
+    knownComputations[requestId] = {
+      callback: callbacks,
+      onMessage: router.onMessage,
```

Comment:
`router` isn't defined here. I don't think you need a `routedMessageHandler()` here anyway. Just set:

```js
onMessage: callbacks.onMessage,
```

You also don't need to set `callback` here, it won't be used.